docs/diagrams/EditActivityDiagram.puml

```diff
@@ -1,16 +1,28 @@
 @startuml
-start
-:User executes edit command;
-
-'Since the beta syntax does not support placing the condition outside the
-'diamond we place it as the true branch instead.
+'https://plantuml.com/activity-diagram-beta
 
+start
+:User enters edit command;
+:AddressBookParser parses the command;
+:EditCommandParser parses the arguments provided;
 if () then ([command is valid])
-    :AddressBookParser parses the command;
-    :EditCommand replaces the person to be edited
-    with a new person object containing the updated
-    data fields;
-else ([else])
+  :All the arguments are initialized in an EditPersonDescriptor object;
+  :New EditCommand object is created;
+  if() then ([invalid index])
+    :Outputs error message;
+    else ([else])
+    :CreateEditedPerson method in EditCommand creates and returns a new person object containing the updated data fields;
+    if() then ([Throws Command Exception])
```

Comment:
change this to the else block.

Old else block will become the new if (Contact is valid)